src/app/ui/header/header.component.html

```diff
@@ -1,14 +1,17 @@
 <header>
   <div id="topbar">
     <div class="container">
+
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <a class="navbar-brand" href="#">
-          <img src="/assets/images/testock-logo.png" alt="" height="25">
-        </a>
+
         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
 
+        <a class="navbar-brand" href="/">
+          <img class="mx-auto d-block" src="/assets/images/testock-logo.png" alt="" height="25">
```

Comment:
why making this move of the logo?
what is `mx-auto`? what is `d-block`? do you have to use them?

Reply:
return the logo to its original place. 
removed `mx-auto` and `d-block` classes